app/messages.py

```diff
@@ -9,7 +9,9 @@
 FIELD_AVAILABLE_TO_MENTOR_IS_INVALID = {
     "message": "Field available_to_mentor" " is not valid."
 }
-
+USERNAME_FIELD_HAS_INVALID_LENGTH={
+    "message": "The name field has to be longer than 1 characters and shorter than 31 characters."
```

Comment:
```suggestion
    "message": f"The name field has to be longer than {NAME_MIN_LENGTH-1} characters and shorter than {NAME_MAX_LENGTH+1} characters."
```
For this you will have to import these variables in this file